examples/translation_example.py

```diff
@@ -32,29 +40,27 @@
     parse_client_args,
     parse_distributed_classifier_args,
 )
-
+from dask.distributed import get_worker
+MAX_OUT_LEN= 256
+MAX_TOKEN_LEN=200
```

Comment:
Maybe move this inside the config so that in future you only have to modify this class for experimentation

Reply:
Moved MAX_TOKEN_LEN as max_words_per_sen in translation config data class.
Need to test the code with latest CrossFit since it handles postprocessing and hence won't require MAX_OUT_LEN variable.